File: python/linear_search.py
array_to_search_through = []
for number in range(1, 1001):
    array_to_search_through.append(number)

def linear_search(value_to_find, array_to_search_through):
    for element in array_to_search_through:
        if element == value_to_find:
            return array_to_search_through.index(element)
    

def linear_search_global(value_to_find, array_to_search_through):
    answer = []
    for element in array_to_search_through:
        if element == value_to_find:
            answer.append(array_to_search_through.index(element))
    return answer

File: js/linearSearch.js
const arrayToSearchThrough = [];
for (let i = 1; i <= 1000; i++) {
    arrayToSearchThrough.push(i);
}

exports.linearSearch = function(valueToFind, arrayToSearchThrough) {
    // your code here
    for (i=0; i< arrayToSearchThrough.length; i++){
        if(arrayToSearchThrough[i]===valueToFind){
            console.log(i)
            return i
        }
    }
    return undefined
};

exports.linearSearchGlobal = function(valueToFind, arrayToSearchThrough) {
    let answer=[]
    for (i=0; i<arrayToSearchThrough.length; i++){
        if (arrayToSearchThrough[i]=== valueToFind){
            answer.push(i)
        }
    }
    console.log(answer)
    return answer
};